recipes-core/odhcpd/odhcpd_git.bb

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2018 Daniel Dickinson <cshored@thecshore.com>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+DESCRIPTION = "OpenWrt DHCP/DHCPv6(-PD)/RA Server & Relay"
+HOMEPAGE = "http://git.openwrt.org/?p=project/odhcpd.git;a=summary"
+LICENSE = "GPL-2.0"
+LIC_FILES_CHKSUM = "file://src/odhcpd.c;beginline=1;endline=13;md5=b5b1da01ca7e1cdd0308c552dc0a1384"
+SECTION = "base"
+DEPENDS = "libubox ubus libnl uci"
+RDEPENDS_${PN} += " base-files-scripts-openwrt "
+
+inherit cmake pkgconfig openwrt openwrt-services openwrt-base-files
+
+SRCREV_odhcpd = "750e457e3000187b85906814a2529ede24775325"
+SRCREV_openwrt = "${OPENWRT_SRCREV}"
+
+SRC_URI = "\
+    git://git.openwrt.org/project/odhcpd.git;name=odhcpd \
+"
+
+S = "${WORKDIR}/git"
+OF = "${S}/openwrt/package/network/services/odhcpd/files"
+
+EXTRA_OECMAKE_append = " -DUBUS=1 -Dlibnl=${STAGING_LIBDIR}/libnl-3.so -Dlibnl-tiny_include_dir=${STAGING_INCDIR}/libnl3"
+
+OECMAKE_C_FLAGS += "-Wno-error=cpp"
```

Comment:
why is this needed ?

Reply:
hmmm....it might not be, that was left over from copy&paste.